docs/guides/voyage/voyage.md

```diff
@@ -97,6 +97,21 @@ Voyages have specific requirements, milestones, and ***require a commitment of 8
 
 ![How it Works](./assets/Voyage_Project_How_it_Works.png)
 
+When your Voyage starts you will see that we have provided your team with your 
+own team channel in Discord. You should use this to communicate and collaborate
+with one another on your project. Access to this channel is restricted to your
+team so it's the best place for open, frank, and respectful communication with
+your teammates. **_You must not create your own team channel_**
```

Comment:
I think 
**_You must not create your own team channel_** 
should be 
**_You must not create your own team server_**

Reply:
Thank you for catching this @cherylli! I've uploaded a new commit and it's ready for your review again.

Reply:
looks good!